doc/api/api_changes.rst

```diff
@@ -11,6 +11,34 @@ sources of the changes you are experiencing.
 For new features that were added to matplotlib, please see
 :ref:`whats-new`.
 
+Changes in 1.5.3
+================
+
+``ax.plot(..., marker=None)`` gives default marker
+--------------------------------------------------
+
+Prior to 1.5.3 kwargs passed to `~matplotlib.Axes.plot` were handled
+in two parts -- default kwargs generated internal to
+`~matplotlib.Axes.plot` (such as the cycled styles) and user supplied
+kwargs.  The internally generated kwargs were passed to the
+`~matplotlib.lines.Line2D.__init__` and the user kwargs were passed to
```

Comment:
That would generate as "The internally generated kwargs were passed to the __init__ and ...", right? It seems like using the full name would be a good idea here (no `~`.)